prisma/migrations/20230219032754_init/migration.sql

```diff
@@ -0,0 +1,59 @@
+-- CreateTable
```

Comment:
Is there a way to name the migration files or are all the names auto generated? Just a thought that it might be easier to understand each migration if they are given names